ParseFacebookUtils/Internal/AuthenticationProvider/tvOS/PFFacebookDeviceAuthenticationProvider.h

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "PFFacebookAuthenticationProvider.h"
+
+@interface PFFacebookDeviceAuthenticationProvider : PFFacebookAuthenticationProvider
```

Comment:
Personally, I hate the name 'device' to refer to tvOS, but seeing as its FBSDKs decision, we really should keep it that way I suppose.